cumulusci/tasks/bulkdata/load.py

```diff
@@ -101,6 +104,9 @@ def _init_options(self, kwargs):
         self.options["drop_missing_schema"] = process_bool_arg(
             self.options.get("drop_missing_schema") or False
         )
+        self.options["set_recently_viewed"] = process_bool_arg(
+            self.options.get("set_recently_viewed") or False
```

Comment:
From slack it sounds like the decision was to change this default to True, right?

Reply:
@davisagli yessir, I am looking at making the change now, it has some repercussions on some unit tests.